aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -13,70 +15,76 @@ use aptos_framework::natives::aggregator_natives::aggregator_v2::{
 use aptos_language_e2e_tests::executor::ExecutorMode;
 use proptest::prelude::*;
 
+const STRESSTEST_MODE: bool = false;
+
 const EAGGREGATOR_OVERFLOW: u64 = 0x02_0001;
 const EAGGREGATOR_UNDERFLOW: u64 = 0x02_0002;
 
 const DEFAULT_EXECUTOR_MODE: ExecutorMode = ExecutorMode::SequentialOnly;
 
 fn setup(
     executor_mode: ExecutorMode,
-    aggregator_execution_enabled: bool,
+    aggregator_execution_mode: AggregatorMode,
     txns: usize,
 ) -> AggV2TestHarness {
-    initialize(
-        common::test_dir_path("aggregator_v2.data/pack"),
-        executor_mode,
-        aggregator_execution_enabled,
-        txns,
-    )
+    let path = common::test_dir_path("aggregator_v2.data/pack");
+    match aggregator_execution_mode {
+        AggregatorMode::EnabledOnly => initialize(path, executor_mode, true, txns),
+        AggregatorMode::DisabledOnly => initialize(path, executor_mode, false, txns),
+        AggregatorMode::BothComparison => {
+            initialize_enabled_disabled_comparison(path, executor_mode, txns)
+        },
+    }
 }
 
 #[cfg(test)]
 mod test_cases {
     use super::*;
-    use test_case::test_case;
 
-    #[test_case(true)]
-    #[test_case(false)]
-    fn test_copy_snapshot(execution_enabled: bool) {
-        let mut h = setup(DEFAULT_EXECUTOR_MODE, execution_enabled, 1);
+    #[test]
+    fn test_copy_snapshot() {
+        let mut h = setup(DEFAULT_EXECUTOR_MODE, AggregatorMode::BothComparison, 1);
         let txn = h.verify_copy_snapshot();
-        assert_abort!(h.harness.run(txn), EAGGREGATOR_FUNCTION_NOT_YET_SUPPORTED);
+        h.run_block_in_parts_and_check(BlockSplit::Whole, vec![(
+            EAGGREGATOR_FUNCTION_NOT_YET_SUPPORTED,
+            txn,
+        )]);
     }
 
-    #[test_case(true)]
-    #[test_case(false)]
-    fn test_copy_string_snapshot(execution_enabled: bool) {
-        let mut h = setup(DEFAULT_EXECUTOR_MODE, execution_enabled, 1);
+    #[test]
+    fn test_copy_string_snapshot() {
+        let mut h = setup(DEFAULT_EXECUTOR_MODE, AggregatorMode::BothComparison, 1);
         let txn = h.verify_copy_string_snapshot();
-        assert_abort!(h.harness.run(txn), EAGGREGATOR_FUNCTION_NOT_YET_SUPPORTED);
+        h.run_block_in_parts_and_check(BlockSplit::Whole, vec![(
+            EAGGREGATOR_FUNCTION_NOT_YET_SUPPORTED,
+            txn,
+        )]);
     }
 
-    #[test_case(true)]
-    #[test_case(false)]
-    fn test_snapshot_concat(execution_enabled: bool) {
-        let mut h = setup(DEFAULT_EXECUTOR_MODE, execution_enabled, 1);
+    #[test]
+    fn test_snapshot_concat() {
+        let mut h = setup(DEFAULT_EXECUTOR_MODE, AggregatorMode::BothComparison, 1);
         let txn = h.verify_string_concat();
-        assert_success!(h.harness.run(txn));
+        h.run_block_in_parts_and_check(BlockSplit::Whole, vec![(0, txn)]);
```

Comment:
can we have `const SUCCESS = 0` or something similar here or at the start of the file, so it is clear it is an abort/success status

Reply:
we do, I just missed using it here